packages/truffle-db/src/loaders/artifacts/index.ts

```diff
@@ -167,6 +179,22 @@ export class ArtifactsLoader {
       ({ id }) => ({ id })
     );
 
+    const nameRecords = await Promise.all(
+      contractObjects.map(async (contract, index) => {
+        let nameRecordObject = {
+          name: contract.name,
+          type: "Contract",
+          resource: {
+            id: contractIds[index].id
+          }
+        };
+
+        return nameRecordObject as NameRecordObject;
+      })
+    );
```

Comment:
This looks like a regular projection, is `Promise.all` necessary?

Reply:
Nope, I'll fix it!